TriblerGUI/core_manager.py

```diff
@@ -44,8 +49,45 @@ def shutdown(session, *_):
 
     sys.path.insert(0, base_path)
 
+    def get_keyring_password(service, username):
+        child_pipe.send(('get_keyring_password', {'service': service, 'username': username}))
+        return child_pipe.recv()
+
+    def set_keyring_password(service, username, password):
+        child_pipe.send(('set_keyring_password', {'service': service, 'username': username, 'password': password}))
+        child_pipe.recv()
+
+    def patch_wallet_methods():
+        BitcoinWallet.get_wallet_password = lambda _: get_keyring_password('tribler', 'btc_wallet_password')
+        BitcoinWallet.set_wallet_password = lambda _, password: set_keyring_password('tribler',
+                                                                                     'btc_wallet_password', password)
+
+    def patch_iom_methods():
```

Comment:
This is already some code for our internet-of-money integration. This will be my next PR.